core/includes/theme.inc

```diff
@@ -2446,7 +2446,9 @@ function theme_head_tag($variables) {
   $element = $variables['element'];
   $attributes = isset($element['#attributes']) ? backdrop_attributes($element['#attributes']) : '';
   if (!isset($element['#value'])) {
-    return '<' . $element['#tag'] . $attributes . " />\n";
+    if (isset($element['#tag'])) {
```

Comment:
nice catch, but can be single ligned with the upper condition

Reply:
moreover, using `!empty()` instead of `isset()` can prevent empty value of `#tag`.

Reply:
@opi That's what I did at first, but then it gets sent into the `else` section below and outputs other empty elements -- `<><>` instead of `< />< />`

Reply:
`!empty` sounds like a good idea then.